crates/sp-domains-fraud-proof/src/verification.rs

```diff
@@ -459,6 +468,43 @@ where
                 Err(VerificationError::InvalidProof)
             }
         }
+        InvalidBundlesFraudProofType::IllegalTx(extrinsic_index, storage_proof) => {
+            let bundle_body = get_fraud_proof_verification_info(
+                bad_receipt.consensus_block_hash.into(),
+                FraudProofVerificationInfoRequest::DomainBundleBody {
+                    domain_id: invalid_bundles_fraud_proof.domain_id,
+                    bundle_index: invalid_bundles_fraud_proof.bundle_index,
+                },
+            )
+            .and_then(FraudProofVerificationInfoResponse::into_bundle_body)
+            .ok_or(VerificationError::FailedToGetDomainBundleBody)?;
+
+            // Make host call for check extrinsic in single context
+            let check_extrinsic_result = get_fraud_proof_verification_info(
+                bad_receipt.consensus_block_hash.into(),
+                FraudProofVerificationInfoRequest::CheckExtrinsicsInSingleContext {
+                    domain_id: invalid_bundles_fraud_proof.domain_id,
+                    domain_block_number: bad_receipt_parent.domain_block_number.saturated_into(),
+                    domain_block_hash: bad_receipt_parent.domain_block_hash.into(),
+                    domain_block_state_root: bad_receipt_parent.final_state_root.into(),
+                    extrinsics: bundle_body,
```

Comment:
We do not need to verify the whole bundle body but instead should stop at `extrinsic_index`